sql/01-enable_versioning.sql

```diff
@@ -228,6 +228,11 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
-DROP EVENT TRIGGER IF EXISTS _ver_abort_on_drop_of_versioned_table;
-CREATE EVENT TRIGGER _ver_abort_on_drop_of_versioned_table
+-- This is only needed on databases which ever installed a
+-- development version of table_version, which at one point
+-- created a trigger named "_ver_abort_drop_of_versioned_table"
+DROP EVENT TRIGGER IF EXISTS _ver_abort_drop_of_versioned_table;
+
+DROP EVENT TRIGGER IF EXISTS _ver_avoid_drop_of_versioned_table;
```

Comment:
I like more `_ver_abort_drop_of_versioned_table` than `_ver_avoid_drop_of_versioned_table`

Reply:
I think we don't need to worry about backwards compatibility of changes in this patch. As I wrote above, there is no prod or any other significant deployment done from master branch.